docs/Getting-Started-TPU-VM.md

```diff
@@ -85,18 +85,37 @@ the VM. That's explained down below in the [Running Levanter GPT-2](#running-lev
 ## Running Levanter GPT-2
 Now that you have a TPU VM instance, you can follow the [Getting Started](Getting-Started-Training.md) steps, but here are a few shortcuts:
 
-### Launch a GPT-2 Small in unattended mode (using nohup)
+### Launch a GPT-2 Small in unattended mode
```

Comment:
we should tell people to install docker and have it running on their development machine

Reply:
Done.